src/solver/conv_winoRxS_f2x3.cpp

```diff
@@ -480,7 +485,7 @@ bool ConvBinWinogradRxSf2x3::IsApplicable(const ConvolutionContext& params) cons
 {
     if(!params.Is2d())
         return false;
-    if(!params.IsFp32())
+    if(!(params.IsFp32() || params.IsFp16()))
```

Comment:
Same coverage as FP32, none new limitations like `(c % 2 == 0)`?

Reply:
Yes, same as for FP32.